packages/package-manager/src/NpmPackageManager.ts

```diff
@@ -0,0 +1,179 @@
+import JsonFile from '@expo/json-file';
+import spawnAsync, { SpawnOptions } from '@expo/spawn-async';
+import ansiRegex from 'ansi-regex';
+import assert from 'assert';
+import fs from 'fs';
+import npmPackageArg from 'npm-package-arg';
+import path from 'path';
+import rimraf from 'rimraf';
```

Comment:
we are trying to get rid of `fs-exra` in favor of `fs/promise`, but we are still using `rimraf`?

Reply:
Same as https://github.com/expo/expo-cli/pull/4346#discussion_r1008043056:
> Yeah, I think @dsokal mentioned getting rid of rimraf. We can definitely swap rimraf for just fs and use async methods. Can follow up in a later PR after https://github.com/expo/expo/pull/18576

We can definitely do that in a follow-up PR :). For now, I left in the same as it was, but it's an async method, we can swap it out.